nixos/modules/services/logging/logrotate.nix

```diff
@@ -155,7 +155,7 @@ in
     systemd.services.logrotate = {
       description = "Logrotate Service";
       wantedBy = [ "multi-user.target" ];
-      startAt = "*-*-* *:05:00";
```

Comment:
Wait, why doesn't this use the configured frequency?

Reply:
That isn't actually how `logrotate` works, though a reasonable assumption if you're unfamiliar with it. When `logrotate` runs it will execute all outstanding rotations so you want it to run hourly usually.

Reply:
The existing `startAt` value runs it at 5 minutes past every hour, if I'm not wrong. Hourly will move it to minute 0.

Reply:
We should leave it the way it is now. Users who have many jobs running at the same time can override `systemd.services.logrotate.startAt` to suit there needs of they have issues (which they shouldn't).

I think it is time to click merge. Any objections?

Reply:
wfm.